### PR TITLE
refactor: optimize performance of mapToRange400()

### PR DESCRIPTION
**Pull request by Bexin3**
The function has been made much faster

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -144,9 +144,9 @@ void loop()
 
 int mapToRange400(int value, int min, int max)
 {
-  float multiplier = (value - min) * 1.0 / (max - min);
-
-  return min(max(round(multiplier * 400), 0), 400);
+  uint16_t finalval = (value-min)*400/(max-min);
```
Comment (minisbett) [inline src/main.cpp]: Could you change the name of the variable to `new`?

Comment (minisbett) [reply src/main.cpp]: And also put spaces between the operators and operands

**Review by minisbett — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 1 above)
Comment (Bexin3) [reply src/main.cpp]: sure

**Comment by minisbett** [comment]
Closed as this code has long been refactored anyways.

**Pull request closed without merging**